manifests/plugin_panels.pp

```diff
@@ -0,0 +1,22 @@
+# panels-specific functionality.
+# TODO: probably should move this file into its own puppet module
+# TODO: need to make config handling more generic, this should work with ANY plugin, and not be panels-specific
+
+class uber::plugin_panels (
+  $git_repo = "https://github.com/magfest/panels",
+  $git_branch = "master",
```

Comment:
One other thing: It's also possible to remove panels_plugin from profile_rams_full_stack and include it in a magfest-specific file like event-prime.yaml, by doing the following:

```
---
classes:
- roles::uber_server
- uber::panels_plugin
```

The question then really becomes "should panels be a default plugin for all RAMS installations"?  If yes, then it's ok to put it in profile_rams_full_stack.  If no, then it would be better to put it as I have above in a magfest-specific .yaml file.